config/samples/crowdstrike_v1alpha1_falconsensor.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: falcon.crowdstrike.com/v1alpha1
+kind: FalconSensor
+metadata:
+  name: falcon-helm
+spec:
+  # Default values copied from <project_dir>/helm-charts/falcon-sensor/values.yaml
+  falcon:
+    aid: null
+    apd: null
+    aph: null
+    app: null
+    assert: null
+    billing: null
+    cid: null
+    feature: null
+    memfail_every_n: null
+    memfail_grace_period: null
+    message_log: null
+    provisioning_token: null
+    tags: null
+    trace: null
+    update: null
```

Comment:
If these are the default/unchanged values, why copy them?

Reply:
because it is a sample customized CRD... and you might not dig into the helm-chart values. This makes it easier on the user to just modify and run without having to dig deeper.